bindings/pyroot/pythonizations/test/rbatchgenerator_completeness.py

```diff
@@ -1,14 +1,16 @@
 import unittest
 import os
 import ROOT
+# from ROOT import RVec
 import numpy as np
 from random import randrange
-
+ROOT.gInterpreter.Declare("#include <ROOT/RVec.hxx>")
```

Comment:
I find this to be scary, is this actually required? Why so? I believe we should try to revert to `from ROOT import RVec` or anyway just `ROOT.RVec` when needed in the tests